pages/profile/treemapper/my-species.tsx

```diff
@@ -1,20 +1,23 @@
 import Head from 'next/head';
-import React, { ReactElement } from 'react';
+import React, { ReactElement, useContext } from 'react';
 import UserLayout from '../../../src/features/common/Layout/UserLayout/UserLayout';
 import MySpecies from '../../../src/features/user/TreeMapper/MySpecies';
 import { serverSideTranslations } from 'next-i18next/serverSideTranslations';
 import { useTranslation } from 'next-i18next';
+import { UserPropsContext } from '../../../src/features/common/Layout/UserPropsContext';
+import AccessDeniedLoader from '../../../src/features/common/ContentLoaders/Projects/AccessDeniedLoader';
 
 interface Props {}
 
 export default function MySpeciesPage({}: Props): ReactElement {
   const { t } = useTranslation('me');
+  const { user } = useContext(UserPropsContext);
   return (
     <UserLayout>
       <Head>
         <title>{t('mySpecies')}</title>
       </Head>
-      <MySpecies />
+      {user.type === 'tpo' ? <MySpecies /> : <AccessDeniedLoader />}
```

Comment:
`AccessDeniedLoader` is not translated, but I don't mind.

Reply:
Requested translation at https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1672#discussion_r1110984590